WooCommerce/Classes/ViewRelated/Dashboard/Settings/About/AboutViewController.swift

```diff
@@ -267,6 +269,51 @@ extension AboutViewController: UITableViewDelegate {
     }
 }
 
+// MARK: - Crash Debug Menu
+//
+extension AboutViewController {
+
+    fileprivate var crashDebugMenuGestureRecognizer: UITapGestureRecognizer {
```

Comment:
`private` is good enough in this context. 🙂 

```suggestion
    private var crashDebugMenuGestureRecognizer: UITapGestureRecognizer {
```